var class_py_a_r_t_o_s_1_1artos__wrapper_1_1_lib_a_r_t_o_s_exception =
[
    [ "__init__", "class_py_a_r_t_o_s_1_1artos__wrapper_1_1_lib_a_r_t_o_s_exception.html#a39706f717d9574b9fa29663c7db14c09", null ],
    [ "__repr__", "class_py_a_r_t_o_s_1_1artos__wrapper_1_1_lib_a_r_t_o_s_exception.html#a89a419eac2c43f79d9288a88bfe39401", null ],
    [ "__str__", "class_py_a_r_t_o_s_1_1artos__wrapper_1_1_lib_a_r_t_o_s_exception.html#aa2e1fc051e7afaef8c4e6f0ccc6b052f", null ],
    [ "errcode", "class_py_a_r_t_o_s_1_1artos__wrapper_1_1_lib_a_r_t_o_s_exception.html#aa7a52dfb4a3d460a995774536e5bc8e7", null ],
    [ "errmsg", "class_py_a_r_t_o_s_1_1artos__wrapper_1_1_lib_a_r_t_o_s_exception.html#a843791aa61809753494940763f3c9bc7", null ]
];